gigascience/gigadb-website
# New Version Alert pop up closes

# Pull request for issue: #1825

This is a pull request for the following functionalities:

* The button labeled `Continue to view old version` closes the popup instead of reloading the page

## How to test?

To test in localhost it would be necessary to have a dataset page which is an "old version". I'm not aware of any

However the code update is straightforward and I expect it to work

To visualize the popup I temporarily added `<?php $this->renderPartial('_connection_IsPreviousVersionOf'); ?>` somewhere in the file `protected/views/dataset/view.php`

## How have functionalities been implemented?

The `Continue to view old version` used to reload the page, it's been changed to simply close the modal, i.e.

`<a href="#" class="btn btn-default" data-dismiss="modal">Continue to view old version</a>`

## Any issues with implementation?

--

## Any changes to automated tests?

This test is failing but it's unrelated to the change `admin Dashboard: the currently deployed version of GigaDB is shown on the admin dashboard`
